docs/csharp/programming-guide/nullable-types/handling-null-and-operators.md

```diff
@@ -0,0 +1,159 @@
+# Null coalescing and operators
```

Comment:
We need to add a metadata block. Here's the block I would add:

--
title: "Null coalescing and null conditional operators (C# Programming Guide)"
description: "learn to use the ?? and ?. operators to avoid NullReferenceExceptions in your application"
ms.date: 04//04/2018
ms.prod: .net
ms.technology: 
  - "devlang-csharp"
ms.topic: "article"
helpviewer_keywords: 
  - "?. [C#]"
  - "?? [C#]"
  - "C# language, null conditional, null coalescing"
  - "types [C#], nullable"
author: "jfversluis"
ms.author: "wiwagn"
---